extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -71,26 +65,22 @@ export class WindowPortEmulator {
    * @param {!Window} target
    */
   constructor(win, origin, target) {
-    /** @const {!Window} */
-    this.win = win;
-    /** @private {string} */
+    /** @const @private {!Window} */
+    this.win_ = win;
+    /** @const @private {string} */
     this.origin_ = origin;
-    /** @private @const {!Window} */
+    /** @const @private {!Window} */
     this.target_ = target;
   }
 
   /**
    * @param {string} eventType
-   * @param {function(!Event):undefined} handler
+   * @param {function(!Event):*} handler
    */
   addEventListener(eventType, handler) {
-    this.win.addEventListener('message', e => {
-      if (
-        e.origin == this.origin_ &&
-        e.source == this.target_ &&
-        getData(e)['app'] == APP
-      ) {
-        handler(e);
+    this.win_.addEventListener('message', event => {
+      if (event.origin == this.origin_ && event.source == this.target_) {
```

Comment:
What happened to the `event.data` check? Is it no longer needed? Why?

Reply:
[I moved it to `Messaging`](https://github.com/ampproject/amphtml/pull/24797/files/6a9fa749b68662e8a3916d227193aedc6a54fda8#diff-66a42c891e7eab33fb65af559d832e1dR296) - since this class is meant to be an implementation of `MessagePort`, it seemed wrong to inspect the inside of the message on that level.